app/adapters/application.js

```diff
@@ -0,0 +1,5 @@
+import DS from 'ember-data';
+
+export default DS.FixtureAdapter.extend({
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.